src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let getSumCalls = 0;
```

Comment:
`getSumCalls` sounds more like a function name
```suggestion
  let sumCalls = 0;
```